lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -2,14 +2,21 @@ package MusicBrainz::Server::Form::User::EditProfile;
 
 use HTML::FormHandler::Moose;
 use List::MoreUtils qw( any all );
-use MusicBrainz::Server::Form::Utils qw( language_options select_options_tree );
+use MusicBrainz::Server::Form::Utils qw( language_options select_options_tree validate_username);
```

Comment:
Nitpick: space is missing before the closing `)` like the rest of the imports (same problem in lib/MusicBrainz/Server/Form/User/Register.pm).